### DISCUSSION TITLE
NavigationBurger with sub-menu items

### DISCUSSION THREAD

**Discussion by Iain-g** [flyntwp/flynt#473] [Q&A] [answered]
Hi folks, I'm just experimenting with Flynt and really like it so far - I'd like to stick with Flynt's approach of components throughout if I can. I would normally rely in something like mmenu.js to do my mobile nav but wondering if anyone one knows a good tutorial that would teach me how to adapt the existing NavigationBurger component to show sub-menu items like the screenshot? Sorry, I know its probably a bit of a basic question to some people, but I'm keen to learn.
<img width="642" alt="Screenshot 2023-01-04 at 12 15 29" src="https://user-images.githubusercontent.com/10251588/210553972-545e6493-f956-4a17-b49d-27cbc987d952.png">

**Comment by steffenbew** [comment, marked as answer]
Actually, I don't feel like there's an easy answer here. This would need custom development in any case. We usually like to build mobile menus with dropdown accordion style submenu items. This can be achieved with a bit of JavaScript code. 

What you're showing looks to be more like a sliding subpage of a menu, which is more complex to build. If you're comfortable with mmenu.js, I suggest trying to use that to build a customised navigation component. If you want to learn how to do it from scratch, you might check out the following example with fancy animations: https://tympanus.net/codrops/2015/11/17/multi-level-menu/

**Reply by Iain-g** [reply]
OK thanks @steffenbew , I'll try one of those options, just wanted to check I wasn't missing something obvious.
